flang/include/flang/Optimizer/Transforms/Passes.td

```diff
@@ -179,4 +179,13 @@ def CharacterConversion : Pass<"character-conversion"> {
   ];
 }
 
+def ExternalNameConversion : Pass<"external-name-interop", "mlir::ModuleOp"> {
+  let summary = "Convert name for external interoperability";
+  let description = [{
+    Demangle FIR internal name and mangle them for external interoperability.
+  }];
+  let constructor = "::fir::createExternalNameConversionPass()";
+  let dependentDialects = [ "fir::FIROpsDialect", "mlir::LLVM::LLVMDialect" ];
```

Comment:
Don't forget to add OpenMP, OpenACC, if that was your plan here.

Reply:
Thanks I'll add them.